wrapper/src/main/java/software/amazon/jdbc/hostlistprovider/monitoring/ClusterTopologyMonitorImpl.java

```diff
@@ -503,7 +510,7 @@ protected List<HostSpec> openAnyConnectionAndUpdateTopology() {
 
       // open a new connection
       try {
-        conn = this.pluginService.forceConnect(this.initialHostSpec, this.monitoringProperties);
+        conn = this.connectionService.createAuxiliaryConnection(this.initialHostSpec, this.monitoringProperties);
```

Comment:
Any chance we rename the method to a simpler one? Like openConnection() or just open()

Reply:
I'm fine with it, my only concern is that it might not be clear whether this method creates a full ConnectionWrapper, a pure target driver connection, or a "helper" connection created with forceConnect.